multisite-directory.php

```diff
@@ -246,6 +264,16 @@ public static function dashboard_glance_items ($items) {
         return $items;
     }
 
+    /**
+     * Redirects single site post to the actual site
+     */
+    public static function template_redirect () {
+        if (is_singular(Multisite_Directory_Entry::name)) {
```

Comment:
I like the idea of more quickly navigating from directory entry to the site itself, but this redirect needs to be optional. Part of the point of having the directory entries be posts, themselves, is for the Network Admin to write long-form descriptions of the sites in the network. With this, you are *unconditionally* redirecting all page views of the entry itself to the site that the entry refers to. That isn't the intended purpose of the plugin.

The best way to do this is to store a `post_meta` value in the entry itself that is then checked on this line to see if a redirect should or should not happen, and an option in the main blog's `options` table to check whether this should be active by default or not. (It should default to the same behavior before this patch, meaning it should be off by default.) That way Network Admins can turn on or off the redirect for each individual site in their Directory and can configure the default for new sites.

This is a bit more work, admittedly, and so you may want to remove these changes from #19 and submit a new pull request with these changes. In the mean time, you can [open an issue](https://github.com/meitar/multisite-directory/issues/new) requesting this specific feature so we have a place to hash it out, if needed.